apps/api/src/domain/models/user-management/user/entities/user/coscrad-user.entity.ts

```diff
@@ -84,6 +86,10 @@ export class CoscradUser extends Aggregate {
         this.authProviderUserId = authProviderUserId;
     }
 
+    getName(): MultilingualText {
+        return buildMultilingualTextWithSingleItem(this.username);
```

Comment:
This makes me wonder whether we really need a `name` for all aggregates, or just `Categorizables`.